homeassistant/components/deluge/sensor.py

```diff
@@ -69,6 +93,22 @@ class DelugeSensorEntityDescription(SensorEntityDescription):
         state_class=SensorStateClass.MEASUREMENT,
         value=lambda data: get_state(data, UPLOAD_SPEED),
     ),
+    DelugeSensorEntityDescription(
+        key=PROTOCOL_TRAFFIC_UPLOAD_SPEED,
+        translation_key="protocol_traffic_upload_speed",
+        device_class=SensorDeviceClass.DATA_RATE,
+        native_unit_of_measurement=UnitOfDataRate.KILOBYTES_PER_SECOND,
+        state_class=SensorStateClass.MEASUREMENT,
+        value=lambda data: get_state(data, PROTOCOL_TRAFFIC_UPLOAD_SPEED),
+    ),
+    DelugeSensorEntityDescription(
+        key=PROTOCOL_TRAFFIC_DOWNLOAD_SPEED,
+        translation_key="protocol_traffic_download_speed",
```

Comment:
Why is it named protocol_* when the api calls it dh_*? Seems bad to rename here. (Can be translated to some better name still)

Reply:
the API calls it `dht_upload_rate` and `dht_download_rate`, do you think i should rename the translation key to match? i was trying to match what the UI calls it :

![image](https://github.com/user-attachments/assets/9d2a7832-5aff-4664-b9bc-1b2650243f5c)

Reply:
The data keys should match the API. Translation is used to translate api info to human info. So yes.

Reply:
> The data keys should match the API. Translation is used to translate api info to human info. So yes.

i renamed the static variables in `const.py` to match and updated the sensor definitions in `sensor.py` (for all 4) to refer to the static variables in `const.py`.  I didn't just rename the existing translation keys UPLOAD_SPEED versus `upload_rate` because i didn't want to add even more breaking changes